setup.py

```diff
@@ -94,16 +94,11 @@ def build_extension(self, extension):
     OpenageExtension('openage.util.fslike.cpp', sources=['openage/util/fslike/cpp.pyx'])
 ]
 
-def get_version():
-    with open('openage_version', 'r') as fp:
-        version = fp.read().strip()
-        if version.startswith('v'):
-            version = version[1:]
-        return version
+version = __import__('openage').__version__
```

Comment:
Are you sure this doesn't fail when the project was not compiled yet?
The `openage` import will not succeed if the `libopenage` wasn't built.